util/tar.go

```diff
@@ -138,11 +150,16 @@ func sanitizedName(filename string) string {
 
 // GetFileContentType returns the real file type
 func GetFileContentType(file string) (string, error) { // Open File
+	// #nosec G304 -- The file is just opened to detect the content type
 	f, err := os.Open(file)
```

Comment:
it is possible that using `filepath.Clean` would be enough to satisfy the warning

Reply:
Fixed, thanks!